packages/video_player/lib/video_player.dart

```diff
@@ -32,13 +32,34 @@ class DurationRange {
 }
 
 class VideoPlayerValue {
+  /// The total duration of the video.
+  ///
+  /// Before the [VideoPlayerController] is initialized this is null.
```

Comment:
[nit] Maybe `/// Is null iff [initialized] as false.`

Reply:
Went for
```
  /// Is null when [initialized] is false.
```